project-metrics/metrics_service/metrics/travis_greenness.py

```diff
@@ -14,10 +14,12 @@ class TravisGreennessMetric(base.PercentageMetric):
 
   def _score_value(self, percentage: float) -> models.MetricScore:
     if percentage < 0.6:
+      return models.MetricScore.CRITICAL
+    elif percentage < 0.74:
```

Comment:
I thought that was a finger slip at first 🙃

Reply:
Yeah weird threshold but w/e